components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
Hmmm.... definitely sounds like an error, but I guess I'm not sure who's at fault. I'd think if `Image` is the default and `Fileset` is passed, `Fileset` should be used without error.